spring-boot-modules/spring-boot-custom-starter/greeter-spring-boot-autoconfigure/pom.xml

```diff
@@ -61,6 +61,7 @@
     <properties>
         <spring-boot.version>2.2.6.RELEASE</spring-boot.version>
         <greeter.version>0.0.1-SNAPSHOT</greeter.version>
+        <mockito.version>4.4.0</mockito.version>
```

Comment:
5.11.0?

Reply:
Yes, this is a quick fix to run the build. Once the build passes we can investigate the code changes required with the new version.

Reply:
OK, I will create a separate task for this module.

Reply:
https://team.baeldung.com/browse/JAVA-33818